.github/workflows/sorn-grab.yml

```diff
@@ -0,0 +1,20 @@
+name: "Find SORNs"
+on:
+  schedule:
+    - cron: "11 7 * * 1-5"  # 2:11am eastern, m-f
+  workflow_dispatch:
+
+jobs:
+  find-sorns:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install cf cli
+        run: ./.cloud-gov/install-cf-cli.sh
+      - name: Login and update
+        run: |
+          cf api https://api.fr.cloud.gov
+          cf auth ${{ secrets.CF_USERNAME }} ${{ secrets.CF_PASSWORD }}
+          cf target -o ${{ secrets.CF_ORG }} -s ${{ secrets.CF_SPACE }}
+          cf run-task all_sorns "bundle exec rails federal_register:find_sorns" --name "Find SORNs"
```

Comment:
I'll try adding your `cf:first_instance` code again I removed it when I was just trying to get it to run at all.

Reply:
With the `workflow_dispatch` trigger, I can run the github action by hitting this endpoint.
```
curl -H "Authorization: token MYTOKEN" -d "{\"ref\": \"restart-gh\"}" https://api.github.com/repos/18f/all_sorns/actions/workflows/sorn-grab.yml/dispatches  
```

Reply:
Adding `cf:on_first_instance` broke the task. Reading more about Tasks - they are already run independent of the instances. We don't need to worry about if it is later scaled up to more instances or not.  https://docs.cloudfoundry.org/devguide/using-tasks.html

I think where you have `cf:on_first_instance` in the manifests makes more sense because it a command to the actual instance, they should know how to interpret that rake task.